### PR TITLE
onboard of API kong 

### PR DESCRIPTION
**Pull request by sivakarthikeyanoptit**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ansible/roles/kong-api/defaults/main.yml [hunk 1]
```diff
@@ -5890,3 +5904,597 @@ kong_apis:
...
+    uris: "{{ users_service_prefix }}/v1/users/solutions"
+    upstream_url: "{{ ml_core_service_url }}/v1/users/solutions"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'mlcoreservice'
```
[the remark this answers is not in the document]
Comment (sivakarthikeyanoptit) [reply ansible/roles/kong-api/defaults/main.yml:5936]: sure altamash i will do that

File: ansible/roles/kong-api/defaults/main.yml [hunk 2]
```diff
@@ -1067,7 +1081,7 @@ kong_apis:
 
   - name: freshDesk
     uris: "{{ tickets_prefix }}/v1/create"
-    upstream_url: "{{ freshDesk_url }}/api/v2/tickets"
+    upstream_url: "{{ freshDesk_url }}/v2/tickets"
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:1084]: is this getting changed?

File: ansible/roles/kong-api/defaults/main.yml [hunk 3]
```diff
@@ -5890,3 +5904,597 @@ kong_apis:
...
+      config.whitelist:
+        - 'programAccess'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+      
+  - name: usersSolutions
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5926]: give a meaningful API name @sivakarthikeyanoptit

File: ansible/roles/kong-api/defaults/main.yml (hunk 2 above)
Comment (sivakarthikeyanoptit) [reply ansible/roles/kong-api/defaults/main.yml:1084]: it is not changing before itself it is mentioned as /v2/tickets" only,  so  that i replace it as same

Comment (rjshrjndrn) [reply ansible/roles/kong-api/defaults/main.yml:1084]: @sivakarthikeyanoptit Please keep it the same as the original one.

File: ansible/roles/kong-api/defaults/main.yml [hunk 4]
```diff
@@ -5890,3 +5904,597 @@ kong_apis:
...
+    uris: "{{ users_service_prefix }}/v1/users/programs"
+    upstream_url: "{{ ml_core_service_url }}/v1/users/programs"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'mlcoreservice'
```
[the remark this answers is not in the document]
Comment (sivakarthikeyanoptit) [reply ansible/roles/kong-api/defaults/main.yml]: done

File: ansible/roles/kong-api/defaults/main.yml (hunk 2 above)
Comment (sivakarthikeyanoptit) [reply ansible/roles/kong-api/defaults/main.yml]: @rjshrjndrn done

**Review by SMYALTAMASH — APPROVED** [review]

**Pull request merged**